AWSCore/STS/Info.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
Do we need to add this here?

Reply:
Based on previous PR https://github.com/aws-amplify/aws-sdk-ios/pull/3675#discussion_r666373173 , this was manually removed. I'll remove it here and have also opened a request for codegen to do this